// ant-design-pro/lib/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _Charts = require('./Charts');

var _Charts2 = _interopRequireDefault(_Charts);

var _CountDown = require('./CountDown');

var _CountDown2 = _interopRequireDefault(_CountDown);

var _DescriptionList = require('./DescriptionList');

var _DescriptionList2 = _interopRequireDefault(_DescriptionList);

var _Exception = require('./Exception');

var _Exception2 = _interopRequireDefault(_Exception);

var _FooterToolbar = require('./FooterToolbar');

var _FooterToolbar2 = _interopRequireDefault(_FooterToolbar);

var _GlobalFooter = require('./GlobalFooter');

var _GlobalFooter2 = _interopRequireDefault(_GlobalFooter);

var _HeaderSearch = require('./HeaderSearch');

var _HeaderSearch2 = _interopRequireDefault(_HeaderSearch);

var _NoticeIcon = require('./NoticeIcon');

var _NoticeIcon2 = _interopRequireDefault(_NoticeIcon);

var _PageHeader = require('./PageHeader');

var _PageHeader2 = _interopRequireDefault(_PageHeader);

var _Result = require('./Result');

var _Result2 = _interopRequireDefault(_Result);

var _TagSelect = require('./TagSelect');

var _TagSelect2 = _interopRequireDefault(_TagSelect);

var _AvatarList = require('./AvatarList');

var _AvatarList2 = _interopRequireDefault(_AvatarList);

var _Trend = require('./Trend');

var _Trend2 = _interopRequireDefault(_Trend);

var _NumberInfo = require('./NumberInfo');

var _NumberInfo2 = _interopRequireDefault(_NumberInfo);

var _Ellipsis = require('./Ellipsis');

var _Ellipsis2 = _interopRequireDefault(_Ellipsis);

var _Authorized = require('./Authorized');

var _Authorized2 = _interopRequireDefault(_Authorized);

var _Login = require('./Login');

var _Login2 = _interopRequireDefault(_Login);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var antDesignPro = {
  Charts: _Charts2.default,
  CountDown: _CountDown2.default,
  DescriptionList: _DescriptionList2.default,
  Exception: _Exception2.default,
  FooterToolbar: _FooterToolbar2.default,
  GlobalFooter: _GlobalFooter2.default,
  HeaderSearch: _HeaderSearch2.default,
  NoticeIcon: _NoticeIcon2.default,
  PageHeader: _PageHeader2.default,
  Result: _Result2.default,
  TagSelect: _TagSelect2.default,
  AvatarList: _AvatarList2.default,
  Trend: _Trend2.default,
  NumberInfo: _NumberInfo2.default,
  Ellipsis: _Ellipsis2.default,
  Authorized: _Authorized2.default,
  Login: _Login2.default
}; // !!! This file will be generated by ./tool/create.js auto

exports.default = antDesignPro;
module.exports = exports['default'];
